python/jax/carfac_bench.py

```diff
@@ -179,7 +179,7 @@ def loss_func(
       weights: carfac_jax.CarfacWeights,
       state: carfac_jax.CarfacState,
   ):
-    nap_output, _, _, _, _, _ = carfac_jax.run_segment(
+    nap_output, naps_fibers_output, _, _, _, _ = carfac_jax.run_segment(
```

Comment:
Why add this as a named output (here and below)

Reply:
Not necessary at all...probably left over from testing!!